src/components/form/SearchForm.tsx

```diff
@@ -0,0 +1,244 @@
+      autoComplete="off"
+      onFinish={handleFinish}
+      style={{
+        padding: '20px',
+        background: '#fafafa',
+        borderRadius: '8px',
+        border: '1px solid #e8e8e8',
+      }}
+    >
+      <Row gutter={[16, 0]}>
+        {/* First column - spans 2 rows */}
+        <Col xs={24} sm={12} md={6}>
+          <Form.Item<SearchFormValues>
+            name="name"
+            label={t('form.searchForm.fields.name')}
+            style={{ marginBottom: '16px' }}
+          >
+            <Input allowClear />
+          </Form.Item>
+          <Form.Item<SearchFormValues>
+            name="id"
+            label={t('form.searchForm.fields.id')}
+            style={{ marginBottom: '16px' }}
+          >
+            <Input allowClear />
+          </Form.Item>
+        </Col>
+
+        {/* Second column - first row */}
+        <Col xs={24} sm={12} md={6}>
+          <Form.Item<SearchFormValues>
+            name="host"
+            label={t('form.searchForm.fields.host')}
+            style={{ marginBottom: '16px' }}
+          >
+            <Input allowClear />
+          </Form.Item>
+          {/* Second column - second row */}
+          <Form.Item<SearchFormValues>
+            name="plugin"
+            label={t('form.searchForm.fields.plugin')}
+            style={{ marginBottom: '16px' }}
+          >
+            <Input allowClear />
+          </Form.Item>
+        </Col>
+
+        {/* Third column - first row */}
+        <Col xs={24} sm={12} md={6}>
+          <Form.Item<SearchFormValues>
+            name="path"
+            label={t('form.searchForm.fields.path')}
+            style={{ marginBottom: '16px' }}
+          >
+            <Input allowClear />
+          </Form.Item>
+          {/* Third column - second row */}
+          <Form.Item<SearchFormValues>
+            name="labels"
+            label={t('form.searchForm.fields.labels')}
+            style={{ marginBottom: '16px' }}
+          >
+            <Select
+              mode="multiple"
+              placeholder={t('form.searchForm.placeholders.labels')}
+              allowClear
+              options={labelOptions}
+            />
+          </Form.Item>
+        </Col>
+
+        {/* Fourth column - first row */}
+        <Col xs={24} sm={12} md={6}>
+          <Form.Item<SearchFormValues>
+            name="description"
+            label={t('form.searchForm.fields.description')}
+            style={{ marginBottom: '16px' }}
+          >
+            <Input allowClear />
+          </Form.Item>
+          {/* Fourth column - second row */}
+          <Form.Item<SearchFormValues>
+            name="version"
+            label={t('form.searchForm.fields.version')}
+            style={{ marginBottom: '16px' }}
+          >
+            <Select
+              placeholder={t('form.searchForm.placeholders.version')}
+              allowClear
+              options={versionOptions}
+            />
+          </Form.Item>
```

Comment:
The `filterRoutes / needsClientSideFiltering` function did not read `filters.version` at all.